Formula/opentsdb.rb

```diff
@@ -16,7 +16,7 @@ class Opentsdb < Formula
     sha256 cellar: :any_skip_relocation, high_sierra: "5bcdc828069e124c16e1e6c8b2eb6732d0ef88533c27f60fcbb0bec369aca375"
   end
 
-  deprecate! date: "2020-11-13", because: :does_not_build
+  disable! date: "2022-07-31", because: :does_not_build
```

Comment:
We are able to build `opentsdb` now (#100699) but I am stuck on figuring out why `hbase` LZO isn't working on Monterey (#104233). I could just remove compression or use gzip instead to get that re-enabled.